processing/src/main/java/io/druid/segment/FloatDimensionIndexer.java

```diff
@@ -142,7 +142,7 @@ public float get()
         final Object[] dims = currEntry.getKey().getDims();
 
         if (dimIndex >= dims.length) {
-          return 0.0f;
+          return DimensionHandlerUtils.ZERO_FLOAT;
```

Comment:
The last ask about using ZERO_ constants was particularly about `makeObjectColumnSelector()` methods, because they return Object. This method returns primitive so the change here is pointless (and in some other places in this commit)